test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -1428,3 +1428,89 @@
+  - opcode: createPods
+    countParam: $measurePods
+    podTemplatePath: config/event_handling/add-node-unsched-pod-nodeunschedulable.yaml
+    skipWaitToCompletion: true
+  # Wait for pods to be processed by the scheduler.
+  - opcode: sleep
+    duration: 10s
+  # Create schedulable node with low capacity.
+  - opcode: createNodes
+    count: 1
+    nodeTemplatePath: config/event_handling/add-node-low-capacity.yaml
+  # Created pods blocked using NodeResources plugin.
+  - opcode: createPods
+    countParam: $measurePods
+    podTemplatePath: config/event_handling/add-node-unsched-pod-noderesources.yaml
+    skipWaitToCompletion: true
+  # Wait for unschedulable pods to be processed by the scheduler.
+  - opcode: sleep
+    duration: 10s
+  # Create nodes that will have enough resource capacity for pods blocked by NodeResources plugin.
+  # These nodes will still block the next pods from being scheduled.
+  - opcode: createNodes
+    countParam: $nodes
+    nodeTemplatePath: config/event_handling/add-node-high-capacity.yaml
+  # Wait on barrier for NodeUnschedulable and NodeResources pods to be scheduled.
+  - opcode: barrier
+  # Created pods blocked using InterPodAffinity plugin.
+  - opcode: createPods
+    countParam: $measurePods
+    podTemplatePath: config/event_handling/add-node-unsched-pod-interpodaffinity.yaml
+    skipWaitToCompletion: true
+  # Created pods blocked using NodeAffinity plugin.
+  - opcode: createPods
+    countParam: $measurePods
+    podTemplatePath: config/event_handling/add-node-unsched-pod-nodeaffinity.yaml
+    skipWaitToCompletion: true
+  # Created pods blocked using PodTopologySpread plugin.
+  - opcode: createPods
+    countParam: $measurePods
+    podTemplatePath: config/event_handling/add-node-unsched-pod-podtopologyspread.yaml
+    skipWaitToCompletion: true
+  # Wait for unschedulable pods to be processed by the scheduler.
+  - opcode: sleep
+    duration: 30s
+  # Create nodes that will unblock most of the unschedulable pods.
+  - opcode: createNodes
+    countParam: $nodes
+    nodeTemplatePath: config/event_handling/add-node-unblocker.yaml
+  # Wait on barrier for InterPodAffinity, NodeAffinity and PodTopologySpread pods to be scheduled.
+  - opcode: barrier
+  # Created pods blocked using TaintToleration plugin.
+  - opcode: createPods
+    countParam: $measurePods
+    podTemplatePath: config/event_handling/add-node-unsched-pod-tainttoleration.yaml
+    skipWaitToCompletion: true
+  # Wait for unschedulable pods to be processed by the scheduler.
+  - opcode: sleep
+    duration: 10s
+  # Create nodes that will unblock pods filtered out by TaintToleration plugin.
+  - opcode: createNodes
+    countParam: $nodes
+    nodeTemplatePath: config/event_handling/add-node-with-taint.yaml
+  # Wait on barrier for TaintToleration pods to be scheduled.
+  - opcode: barrier
+  - opcode: stopCollectingMetrics
+  workloads:
+  - name: 100Nodes_500Pods
+    labels: [performance, short]
```

Comment:
It's not that short right now, given all the sleeps. But hopefully we can optimize.